MEETINGS.md

```diff
@@ -8,7 +8,7 @@
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------|----------|
 | 422 | [Basilur]          | [Black]     | [Magic Nights]              | [Sri Lanka] | Ruslan Kazakov  | 19-11-18 |
 | 421 | [Adore Tea]        | [Black]     | [Devonshire Cream]          | [Sri Lanka] | Gary Butler     | 12-11-18 |
-| 420 | Fix this           | [Herbal]    | Fix this                    | [Brazil]    | Fagner Brack    | 09-11-18 |
+| 420 | [Ximango]          | [Herbal]    | [Chimarrao]                 | [Brazil]    | Fagner Brack    | 09-11-18 |
```

Comment:
```suggestion
| 420 |           | [Herbal]    | [Chimarrao]                 | [Brazil]    | Fagner Brack    | 09-11-18 |
```
As [Ximango] and [Chimarrao] are links could you please provide links for them in the second section at the bottom?